lib/handleRequest.js

```diff
@@ -8,7 +8,11 @@ const {
 } = require('./symbols')
 
 function handleRequest (err, request, reply) {
-  if (reply.sent === true) return
+  if (reply.sent === true) {
```

Comment:
If you throw the error like this. What happen if people call the `reply` in hook?